src/ui/projectmanager/db/pm.go

```diff
@@ -108,3 +104,32 @@ func (p *ProjectManager) GetRoles(username string, projectIDOrName interface{})
 
 	return roles
 }
+
+// GetPublic returns all public projects
+func (p *ProjectManager) GetPublic() []models.Project {
```

Comment:
It is called by ProjectManager.GetPublic(), so I think the prefix may declare the return are projects clearly.